app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -34,35 +34,35 @@
     </div>
   </div>
 
-  <div class="media-component-body">
+  <div class="media-component-body" id="left-drawer">
```

Comment:
This is more than the left drawer.

Reply:
Sure, I don't think in this context it's being used informatively, it's just the link to the `aria-controls` value, but perhaps I'm misunderstanding.

Reply:
I'm saying that this is the parent container of the left drawer and contains the media player and the right drawer too.

Reply:
Ah, gotcha.. Yeah, that's not right, I probably just put this in the wrong place, apologies.